web/util.py

```diff
@@ -24,3 +25,12 @@ def flash_redirect(path: str, message: str | None = None, category="info"):
         flash(message, category)
 
     return redirect(path)
+
+
+def upload_file_to_printer(app, file):
+    user_name = request.headers.get("User-Agent", "ankerctl").split(url_for('app_root'))[0]
+
+    with app.svc.borrow("filetransfer") as ft:
+        ft.send_file(file, user_name)
+
+    return
```

Comment:
Read my previous comment on the matter, the `return` is warranted and needed because without it, the function that called this from flask never reached the redirect on successful upload and thus the modal stays on the screen and the user is never told whether or not the upload completed.